runelite-client/src/test/java/net/runelite/client/plugins/screenshot/ScreenshotPluginTest.java

```diff
@@ -245,4 +248,144 @@ public void testHunterLevel2()
 
 		verify(drawManager).requestNextFrameListener(any(Consumer.class));
 	}
+
+	@Test
+	public void testTheCorsairCurseQuestComplete()
+	{
+		Widget questChild = mock(Widget.class);
+		when(client.getWidget(eq(QUEST_COMPLETED_NAME_TEXT))).thenReturn(questChild);
+
+		when(questChild.getText()).thenReturn("You have completed The Corsair Curse!");
+
+		assertEquals("Quest(The Corsair Curse)", screenshotPlugin.parseQuestCompletedWidget());
+
+		WidgetLoaded event = new WidgetLoaded();
+		event.setGroupId(QUEST_COMPLETED_GROUP_ID);
+		screenshotPlugin.onWidgetLoaded(event);
+
+		GameTick tick = new GameTick();
+		screenshotPlugin.onGameTick(tick);
+
+		verify(drawManager).requestNextFrameListener(any(Consumer.class));
```

Comment:
All of this code is basically only testing `parseQuestCompletedWidget() != null` in the context of the calling code, so it can be removed.
```suggestion
```